python-checks/src/main/java/org/sonar/python/checks/IsCloseAbsTolCheck.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.python.checks;
+
+import java.util.Optional;
+
+import org.sonar.check.Rule;
+import org.sonar.plugins.python.api.PythonSubscriptionCheck;
+import org.sonar.plugins.python.api.SubscriptionContext;
+import org.sonar.plugins.python.api.tree.CallExpression;
+import org.sonar.plugins.python.api.tree.Expression;
+import org.sonar.plugins.python.api.tree.Name;
+import org.sonar.plugins.python.api.tree.NumericLiteral;
+import org.sonar.plugins.python.api.tree.RegularArgument;
+import org.sonar.plugins.python.api.tree.Tree;
+import org.sonar.python.tree.TreeUtils;
+
+@Rule(key = "S6727")
+public class IsCloseAbsTolCheck extends PythonSubscriptionCheck {
+
+  private static final String MESSAGE = "Provide the abs_tol parameter when using math.isclose to compare a value to 0";
+
+  @Override
+  public void initialize(Context context) {
+    context.registerSyntaxNodeConsumer(Tree.Kind.CALL_EXPR,
+        ctx -> checkForIsCloseAbsTolArgument(ctx, (CallExpression) ctx.syntaxNode()));
+  }
+
+  private static void checkForIsCloseAbsTolArgument(SubscriptionContext ctx, CallExpression call) {
+    Optional.ofNullable(call.calleeSymbol())
+        .filter(symbol -> "math.isclose".equals(symbol.fullyQualifiedName()) &&
+            anyArgumentIsZero(call) &&
+            TreeUtils.argumentByKeyword("abs_tol", call.arguments()) == null)
```

Comment:
I see the `abs_tol` parameter is keyword-only. That wasn't clear to me initially.
Maybe we could have a test that highlights that, with a call such as `math.isclose(0, x, 1, 2)` and a comment that says we acknowledge that an issue is raised because the call is not legit anyway.